src/main/java/character/Mage.java

```diff
@@ -0,0 +1,59 @@
+package character;
+
+import geometry.Vector2D;
+
+// Class representing a character : mage in the game
+public class Mage extends Entity {
+
+    public Mage() {
+        this(0, 0);
+    }
+
+    /**
+     * Constructor to create a Mage with specified coordinates.
+     * Initializes Mage animations and sets its position.
+     *
+     * @param x The x-coordinate of the Mage.
+     * @param y The y-coordinate of the Mage.
+     */
+    public Mage(double x, double y) {
+        this.setAnimations("mage/");
+        this.coordinates = new Vector2D(x, y);
+    }
```

Comment:
```suggestion
        this.isFacingLeft = false;
    }
```
Otherwise ``getOffset`` could raise an error because of unset variable.